apps/internal/base/internal/storage/storage.go

```diff
@@ -443,6 +424,68 @@ func (m *Manager) writeAppMetaData(AppMetaData AppMetaData) error {
 	return nil
 }
 
+func (m *Manager) RemoveAccount(account shared.Account, clientID string) error {
```

Comment:
lets document what all RemoveAccount does

Reply:
Done : https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/215/commits/16a8e58fbe4c78cd9c807cb83eae2fa35cb07ccb